pkg/ecdsa/retry_test.go

```diff
@@ -0,0 +1,106 @@
+package ecdsa
```

Comment:
Can we reorder functions in that file? Typical structure is:
```
//Actual test scenarios
TestABC() { ... }
TestXYZ() { ... }

// Helpers 
func initializeABC() { ... }
```

Reply:
[6309ed9](https://github.com/keep-network/keep-core/pull/3109/commits/6309ed935475873a8886891e77b35327e040ba93
)